invenio_rdm_records/resources/serializers/bibtex/schema_formats.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""BibTex Schema formaters, helper file for schema.py file."""
+
+
+class BibTexFormatter:
+    """Helper formater class for parsing all the following types.
+
+    publication
+    publication_annotationcollection
+    publication_book
+    publication_section
+    publication_conferencepaper
+    publication_datamanagementplan
+    publication_article
+    publication_patent
+    publication_preprint
+    publication_deliverable
+    publication_milestone
+    publication_proposal
+    publication_report
+    publication_softwaredocumentation
+    publication_taxonomictreatment
+    publication_technicalnote
+    publication_thesis
+    publication_workingpaper
+    publication_other
+    poster
+    presentation
+    dataset
+    image
+    image_figure
+    image_plot
+    image_drawing
+    image_diagram
+    image_photo
+    image_other
+    video
+    software
+    lesson
+    other.
+    """
+
+    def format(name, req_fields, opt_fields):
+        """Basic output formater for BibTexFormatter."""
+        return {
+            "name": name,
+            "req_fields": req_fields,
+            "opt_fields": opt_fields,
+        }
+
+    # def publication():
```

Comment:
What is there todo with all the commented functions? should we open an issue to investigate?

Reply:
Yup there already is an open issue
https://github.com/inveniosoftware/invenio-rdm-records/issues/1231
as this is a cross-serializer problem

Reply:
Perfect, thanks! then we can take this in the new issue